### PR TITLE
finished task35

### PR DESCRIPTION
**Pull request by nait13**

### CODE REVIEW

File: src/main/java/servlet/ChangeLoginUserServlet.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+        getServletContext().getRequestDispatcher("/WEB-INF/updateLogin.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        String userId = req.getParameter("userId");
+        String newLogin = req.getParameter("login");
+
+        if (userId == null || userId.isEmpty() || newLogin == null || newLogin.isEmpty()) {
+            return;
```
Comment (DHrytsuk) [inline src/main/java/servlet/ChangeLoginUserServlet.java]: It's not correct to use `return` in such methods.
What is the result in this case?

File: src/main/java/servlet/ChangeLoginUserServlet.java [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+        }
+
+        int prepareResultSet;
```
Comment (DHrytsuk) [inline src/main/java/servlet/ChangeLoginUserServlet.java]: Why does it here?

File: src/main/java/servlet/ChangeLoginUserServlet.java [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+        try (Connection connection = PostgresDriverManager.getInstance().getConnection();
+             PreparedStatement preparedStatement = connection.prepareStatement("UPDATE users SET login = ? WHERE id = ?"))
+        {
+            preparedStatement.setString(1, newLogin);
+            preparedStatement.setInt(2, Integer.parseInt(userId));
+
+            prepareResultSet = preparedStatement.executeUpdate();
+            if (prepareResultSet > 0) {
+                req.setAttribute("body","Success change login");
+                getServletContext().getRequestDispatcher("/WEB-INF/information.jsp").forward(req,resp);
```
Comment (DHrytsuk) [inline src/main/java/servlet/ChangeLoginUserServlet.java]: Duplicate line of code should be moved to the common place

File: src/main/java/servlet/ChangeLoginUserServlet.java (hunk 1 above)
Comment (nait13) [reply src/main/java/servlet/ChangeLoginUserServlet.java]: Пустая страница=(

Comment (DHrytsuk) [reply src/main/java/servlet/ChangeLoginUserServlet.java]: ANd what about note about `return`?

Comment (nait13) [reply src/main/java/servlet/ChangeLoginUserServlet.java]: Удалил ruturn =)

Comment (DHrytsuk) [reply src/main/java/servlet/ChangeLoginUserServlet.java]: Now, ypur code is not implemented correctly.
It should look like 
```
if (condition)
{
get error view
}
else
{
main logic
}
```

Comment (nait13) [reply src/main/java/servlet/ChangeLoginUserServlet.java]: ok , fix

**Review by DHrytsuk — APPROVED** [review]

**Pull request closed without merging**